pkg/controller/mattermostcluster/constants/constants.go

```diff
@@ -0,0 +1,13 @@
+package constants
+
+const (
+	// OperatorName is a operator name
+	OperatorName = "mattermost-operator"
+	// DefaultAmountOfPods is the default amount of Mattermost pods
+	DefaultAmountOfPods = 2
+	// DefaultMattermostImage is the default Mattermost docker image
+	DefaultMattermostImage        = "mattermost/mattermost-enterprise-edition:5.8.0"
+	DefaultMattermostDatabaseType = "mysql"
```

Comment:
nit: propose being consistent in adding GoDoc to all fields, if we're going to have GoDoc at all in this type.